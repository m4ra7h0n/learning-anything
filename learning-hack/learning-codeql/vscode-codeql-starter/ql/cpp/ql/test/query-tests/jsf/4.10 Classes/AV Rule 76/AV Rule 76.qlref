jsf/4.10 Classes/AV Rule 76.ql
