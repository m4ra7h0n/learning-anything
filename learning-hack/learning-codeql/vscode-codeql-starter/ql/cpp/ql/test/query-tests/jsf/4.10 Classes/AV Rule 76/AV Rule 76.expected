| test.cpp:5:7:5:12 | Class2 | AV Rule 76: A copy constructor and an assignment operator shall be declared for classes that contain pointers to data items or nontrivial destructors. |
| test.cpp:16:7:16:12 | Class3 | AV Rule 76: A copy constructor and an assignment operator shall be declared for classes that contain pointers to data items or nontrivial destructors. |
| test.cpp:33:7:33:12 | Class4 | AV Rule 76: A copy constructor and an assignment operator shall be declared for classes that contain pointers to data items or nontrivial destructors. |
